test/test_suite.py

```diff
@@ -843,3 +844,38 @@ def test_nolength_binary(self):
 @pytest.mark.skip("Spanner doesn't support quotes in table names.")
 class QuotedNameArgumentTest(_QuotedNameArgumentTest):
     pass
+
+
+class StringTest(_StringTest):
+    def test_literal_backslashes(self):
```

Comment:
With the added `render_literal_value` method, `test_literal_backslashes` and `test_literal_quoting` should no longer need to be overridden. Please remove.